app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="public_recipes_container">
+  <h2>Public Recipes</h2>
+  <ul>
+    <% @recipes.each do |recipe| %>
+    <li>
+      <div>
+        <h4><%= link_to recipe.name, user_recipes_path(recipe), class:"links" %></h4>
+        <p>By: <%= recipe.user.name %></p>
+      </div>
```

Comment:
Hi @Lancelot-SO , clicking on the recipe link should navigate to that particular recipe details. Here the page is navigating to the user's recipes. Please change this.